contracts/interfaces/ITrigger.sol

```diff
@@ -41,31 +41,31 @@ abstract contract ITrigger {
   }
 
   /**
-   * @notice Returns array of IDs, where each ID corresponds to a platform covered by this trigger
+   * @notice Returns an array of IDs, where each ID corresponds to a platform covered by this trigger
    * @dev See documentation for mapping of ID numbers to platforms
    */
   function getPlatformIds() external view returns (uint256[] memory) {
     return platformIds;
   }
 
   /**
-   * @dev Executes trigger-specific logic to check if market has been triggered
-   * @return True if trigger condition occured, false otherwise
+   * @dev Executes trigger-specific logic to check if a protection market has been triggered
+   * @return True if the trigger condition occurred, false otherwise
    */
   function checkTriggerCondition() internal virtual returns (bool);
 
   /**
-   * @notice Checks trigger condition, sets isTriggered flag to true if condition is met, and returns the trigger status
-   * @return True if trigger condition occured, false otherwise
+   * @notice Checks the trigger condition, sets isTriggered flag to true if condition is met, and returns the trigger status
+   * @return True if trigger condition occurred, false otherwise
    */
   function checkAndToggleTrigger() external returns (bool) {
-    // Return true if trigger already toggled
+    // Return true if the trigger condition has already been toggled
     if (isTriggered) return true;
 
-    // Return false if market has not been triggered
+    // Return false if the protection market has not been triggered
```

Comment:
This line, line 62, and line 68 should use similar phrasing. My suggestion would be:

- Line 62: "Return true if the trigger has already been toggled"
- Line 65: "Return false if the trigger has not been toggled"
- Line 68: "Return true if the trigger condition was just toggled"

I prefer that phrasing over "protection market has been triggered" since this trigger exists independently of a protection market, and doesn't need to know about it

Reply:
Makes sense.